Common/UnitDefinitions/DoseAreaProduct.json

```diff
@@ -0,0 +1,84 @@
+﻿{
+  "Name": "DoseAreaProduct",
+  "BaseUnit": "GrayPerSquareMeter",
+  "XmlDocSummary": "It is defined as the absorbed dose multiplied by the area irradiated.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Dose_area_product",
+  "BaseDimensions": {
+    "L": 4,
+    "T": -2
+  },
+  "Units": [
+    {
+      "SingularName": "GrayPerSquareMeter",
```

Comment:
I believe it should be named `GraySquareMeter`, since "Per" indicates `Gy/m²` instead of `Gy·m²`.

`GrayMeterSquared` would follow the naming in the wiki, but `GraySquareMeter` is more consistent with UnitsNet naming conventions.